modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/account_selector/AccountSelector.js

```diff
@@ -0,0 +1,363 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon, {ClayIconSpriteContext} from '@clayui/icon';
+import {Col, Row} from '@clayui/layout';
+import ClayLink from '@clayui/link';
+import ClaySticker from '@clayui/sticker';
+import classNames from 'classnames';
+import {fetch} from 'frontend-js-web';
+import PropTypes from 'prop-types';
+import React, {useEffect, useRef, useState} from 'react';
+
+import ServiceProvider from '../../ServiceProvider/index';
+import {
+	CURRENT_ACCOUNT_UPDATED,
+	CURRENT_ORDER_UPDATED,
+} from '../../utilities/eventsDefinitions';
+import Autocomplete from './../autocomplete/Autocomplete';
+import OrdersTable from './OrdersTable';
+
+const ORDERS_HEADLESS_API_ENDPOINT = ServiceProvider.AdminOrderAPI('v1')
+	.baseURL;
+const ACCOUNTS_HEADLESS_API_ENDPOINT = ServiceProvider.AdminAccountAPI('v1')
+	.baseURL;
+
+function formatStickerName(name) {
+	const words = name.split(' ');
+
+	if (words.length === 1) {
+		return words[0].substring(0, 2).toUpperCase();
+	}
+
+	return (words[0].charAt(0) + words[1].charAt(0)).toUpperCase();
+}
+
+function AccountSticker({className, logoURL, name, size}) {
+	return (
+		<ClaySticker className={className} shape="user-icon" size={size}>
+			{logoURL ? (
+				<ClaySticker.Image alt={name} src={logoURL} />
+			) : (
+				formatStickerName(name)
+			)}
+		</ClaySticker>
+	);
+}
+
+function updateRemoteCurrentAccount(id, url, refresh) {
+	const formData = new FormData();
+	formData.append('accountId', id);
```

Comment:
I think we can add a new line here.
```suggestion
	const formData = new FormData();

	formData.append('accountId', id);
```